cypress/e2e/account/selfEditAccountSettings.feature

```diff
@@ -0,0 +1,34 @@
+@regression_test
+@stable_test
+Feature: Account - Edit e-mail in the account setting page on dBC
```

Comment:
remove `dbc` as this functionality is overall same for all namespaces

Reply:
done! also commented out the unstable scenario with the note.